include/silo/common/genome_symbols.h

```diff
@@ -28,14 +28,27 @@ enum GENOME_SYMBOL {
    H,    // A or C or T
    V,    // A or C or G
    N,    // any base
+   FIRST = GAP,
```

Comment:
Why do we actually need `FIRST` and `LAST`?

Reply:
We dont really need `FIRST`. `LAST` is nice to have for the counting of symbols, to make it clearer what is meant. However, we could also extract that from the enum.